.travis.yml

```diff
@@ -1,8 +1,11 @@
 ---
+sudo: required
+services:
+    - docker
 language: python
 python: 2.7
 cache: pip
-install:
-  - "pip install vim-vint==0.3.9"
-script:
-  - "vint -s ."
+install: |
+    pip install vim-vint==0.3.9
+script: |
+    vint -s . && make test
```

Comment:
Yep I can change that to a `||`.
However, as a logical OR this means that if only one of them fail, the Travis build will still be a "pass"